### ISSUE TITLE
Support for job queue?

### ISSUE THREAD

**Issue by harwoodr** [phpMv/ubiquity#333]
I'm looking at frameworks for rest api based game back-end - and for reasons of load management and race condition mitigation, I plan on using a job queue with a single worker.

Are job queues supported by ubiquity?

**Comment by jcheron** [comment]
No, there's nothing on this subject, but contributions are open and welcome.
They contribute to the life of a project.
